### PR TITLE
Update Data Portal-related documentation

### PR DESCRIPTION
**Pull request by aclum**
This PR updates the text for the how to guide and the tutorial. I would like to pair this updating the video tutorial so will leave this in draft for now. It removes extensive information about the workflows in favor of linking to workflow overviews with the goal of keeping the documentation up to date and avoiding issues with synchronizing across guides.

### CODE REVIEW

**Comment by aclum** [comment]
@ssarrafan @emileyfadrosh @eecavanna  @jkelliher-github this is ready for review with updates to the functional search.

**Review by eecavanna — APPROVED** [review]

**Comment by eecavanna** [comment]
Merging this PR branch into the `main` of this repository will not cause any websites to get automatically updated. Once it has been merged in, either @yxu-lanl or I can log into the Read the Docs (i.e. RTD) website and click a button there that causes one of RTD's servers to "pull" the latest content from the `main` branch of this repository and put it on the [RTD-hosted website](https://nmdc-documentation.readthedocs.io/en/latest/howto_guides/portal_guide.html).

Separately from that, I assume y'all will want this content to end up on the new `docs.microbiomedata.org` site, too. In order to put it there, you can make these same changes in the `main/content/home/src` directory in the `docs` repo.

File: docs/_static/images/howto_guides/portal_guide/data_type.png
Comment (emileyfadrosh) [inline docs/_static/images/howto_guides/portal_guide/data_type.png:1]: Let's update to this screenshot instead that shows the current amount of data available for search. 
Screenshot 2024-12-16 at 7 46 40 PM

**Review by emileyfadrosh — APPROVED** [review]
I updated the screenshot for the data type search. Otherwise good to go!

**Pull request merged**